File: Estrutura_Sequencial/Python/Ex1010.py
peca1, npecas1, vpeca1 = map(float, input().split())
peca2, npecas2, vpeca2 = map(float, input().split())

valorP = (npecas1 * vpeca1) + (npecas2 * vpeca2)

print(f"VALOR A PAGAR: R$ {valorP:.2f}")

File: Estrutura_Sequencial/JavaScript/Ex1010.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
var prompt = function(texto) {return lines.shift();};

var [peca1, qnt1, pecav1] = prompt().split(' ').map(parseFloat);
var [peca2, qnt2, pecav2] = prompt().split(' ').map(parseFloat);

total = qnt1 * pecav1 + qnt2 * pecav2;
console.log("VALOR A PAGAR: R$", total.toFixed(2));